test/distributed/_tensor/debug/test_comm_mode_features.py

```diff
@@ -0,0 +1,358 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+# Owner(s): ["oncall: distributed"]
+
+from typing import Any, Dict
+
+import torch
+from torch.distributed._tensor import DeviceMesh
+from torch.distributed._tensor.api import distribute_tensor, DTensor
+from torch.distributed._tensor.debug import CommDebugMode
+from torch.distributed.tensor.parallel import (
+    ColwiseParallel,
+    parallelize_module,
+    RowwiseParallel,
+)
+from torch.testing._internal.common_utils import run_tests
+from torch.testing._internal.distributed._tensor.common_dtensor import (
+    DTensorTestBase,
+    MLPModule,
+    MLPStacked,
+    ModelArgs,
+    NUM_DEVICES,
+    skip_unless_torch_gpu,
+    Transformer,
+    with_comms,
+)
+
+
+c10d_functional = torch.ops.c10d_functional
+
+
+class PrintModuleParamaterShardingTest(DTensorTestBase):
+    # checks if parameter / sharding info is the same as ground truth
+    def check_same_set_of_keys(self, dict1, dict2):
```

Comment:
Other than the class needed to started with Test as Tianyu mentioned, I think each unit test also needed to be started with `test_`.